theme/layout/breakpoints.less

```diff
@@ -1,2 +1,20 @@
 // Breakpoint
 @responsive-breakpoint: 740px;
+
+.smallDeviceOnly(@rules) {
+  @media only screen and (max-width: (@responsive-breakpoint - 1)) {
+    @rules();
+  }
+}
+
+.aboveSmallDevice(@rules) {
+  @media only screen and (min-width: @responsive-breakpoint) {
+    @rules();
+  }
+}
+
+.classicOnly(@rules) {
```

Comment:
This is kind of a weird name, I assumed it had something to do with the classic font stack at first.

Not sure it's meaningful outside of Candidate.  Typically use `largeScreen` or similar in hirer.